app/assets/javascripts/spree/frontend/solidus_paypal_braintree.js

```diff
@@ -1,5 +1,9 @@
 //= require spree/braintree_hosted_form
 
+//= require ../braintree_3_5_0_client
+//= require ../braintree_3_5_0_paypal
+//= require ../braintree_3_5_0_data_collector
+
```

Comment:
I don't think this belongs into this PR. We are loading the client lib from Braintree on demand and don't include this anymore.